oeis.py

```diff
@@ -258,6 +247,53 @@ def A000203(start=0, limit=20):
     return sequence
 
 
+def fibonacci(n):
+    if n < 0:
+        print("Incorrect input")
+    if n <= 1:
+        return 1
+    else:
+        return fibonacci(n - 1) + fibonacci(n - 2)
+
+
+@oeis
+def A000045(start, limit, plot=False):
+
```

Comment:
The A000045 you deleted had a docstring, this one has not, this is not good.

See `oeis.py --list` to see why dostrings are important.